modules/installation-generate-aws-user-infra.adoc

```diff
@@ -14,8 +14,9 @@ only the machines that it will use. You generate and customize the
 [IMPORTANT]
 ====
 The Ignition config files that the installation program generates contain
-certificates that expires after 24 hours. You must complete you cluster
-installation before the certificate expires.
+certificates that expire after 24 hours. You must complete your cluster
+installation and keep the cluster running for 24 hours to be sure that all
```

Comment:
The PR generally LGTM.
Should we say clearer for "keep the cluster running for 24 hours"? I mean: if we Stop a master within 12 hours after installation and do not do https://bugzilla.redhat.com/show_bug.cgi?id=1713292#c8 , the cluster would look still "running", but it would finally hit bug 1713292, for which I'm not sure what doc statement would be proper/clear enough about "keep the cluster running for 24 hours" to avoid bug 1713292? @sttts any suggestion?

Reply:
"keep the cluster running for 24 hours in non-degraded state" @xingxingxia would that help? Maybe in addition "... in order to ensure that the first certificate rotation has finished (replacing the certificates of the installer)".

Reply:
@xingxingxia I've adjusted the wording, let me know what you think now. Thanks!